src/WebJobs.Script.WebHost/Controllers/HostController.cs

```diff
@@ -155,6 +155,40 @@ public IActionResult DrainStatus([FromServices] IScriptHostManager scriptHostMan
             }
         }
 
+        [HttpPost]
+        [Route("admin/host/resume")]
+        [Authorize(Policy = PolicyNames.AdminAuthLevelOrInternal)]
+        public async Task<IActionResult> Resume([FromServices] IScriptHostManager scriptHostManager)
+        {
+            _logger.LogDebug($"Received request to resume a draining host.");
+
+            if (Utility.TryGetHostService(scriptHostManager, out IDrainModeManager drainModeManager))
+            {
+                _logger.LogDebug($"Host status: {scriptHostManager.State.ToString()}. Drain mode enabled: {drainModeManager.IsDrainModeEnabled}");
+
+                if (scriptHostManager.State != ScriptHostState.Running)
+                {
+                    _logger.LogDebug("Host is not running, cannot resume");
```

Comment:
Add more context in the log. What do we mean by "host is not running"

409 = conflict = waiting for the host to be in the right state to resume. We can only except the request in a certain state:

- Drain mode is enabled
- Host is running